app/src/main/java/edu/mit/mitmobile2/emergency/activity/EmergencyContactsActivity.java

```diff
@@ -36,9 +37,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         // as you specify a parent activity in AndroidManifest.xml.
         int id = item.getItemId();
 
-        //noinspection SimplifiableIfStatement
-        if (id == R.id.action_settings) {
-            return true;
+        switch (id) {
+            case R.id.action_settings:
+                return true;
+            case android.R.id.home:
```

Comment:
Instead of finish(), you can just set the parent of the activity in the manifest to MITMainActivity, that way it'll navigate properly to the home activity